tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -312,12 +312,9 @@ def publish(
             "Publishing from a local environment is not recommend and requires to be logged in Airbyte's DockerHub registry, do you want to continue?",
             abort=True,
         )
-    if ctx.obj["modified"]:
-        selected_connectors_and_files = get_modified_connectors(get_modified_metadata_files(ctx.obj["modified_files"]))
-        selected_connectors_names = [connector.technical_name for connector in selected_connectors_and_files.keys()]
```

Comment:
Modified connectors were already being filtered out in the previous step, so the `selected_connectors_and_files` ctx already does this.

This was overwriting the other filters in the publish step